config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get 'bookings/new'
```

Comment:
No need of a bookings/new route or view since it will be triggered on the trips/show view
```suggestion
```